Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM python:3.6
+# Set up code directory
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
+
+RUN apt-get update && apt-get install -y libssl-dev
+RUN apt-get update && apt-get install -y python3-pip
+RUN apt-get update && apt-get install -y pandoc
+
+#Copy the application py-evm to the /usr/src/app folder
+COPY . /usr/src/app
```

Comment:
@divyanks I'm not proficient with dockerfiles but can you explain this copy to me. The comment doesn't make much sense to me. As far as I can tell, we  aren't copying `py-evm` anywhere which doesn't even exist at this point since it is installed via `pip3`  a bit later. Am I reading this wrong?